src/abstractdatatable/abstractdatatable.jl

```diff
@@ -437,12 +437,13 @@ end
 Indexes of complete cases (rows without null values)
 
 ```julia
-completecases(dt::AbstractDataTable)
+completecases(dt::AbstractDataTable[, indices])
 ```
 
 **Arguments**
 
 * `dt` : the AbstractDataTable
+* `indices` : which columns to check for `Null`s
```

Comment:
Have a look at `nonunique` for an example of how it's usually documented, I think it's more detailed than that. Also need to say that all columns are used by default. For consistency wit that function, should also accept a single symbol or index.